rele/client.py

```diff
@@ -67,11 +67,14 @@ def create_subscription(self, subscription, topic):
         topic_path = self._client.topic_path(self._gc_project_id, topic)
 
         with suppress(exceptions.AlreadyExists):
-            self._client.create_subscription(
-                name=subscription_path,
-                topic=topic_path,
-                ack_deadline_seconds=self._ack_deadline,
-            )
+            try:
+                self._client.create_subscription(
+                    name=subscription_path,
+                    topic=topic_path,
+                    ack_deadline_seconds=self._ack_deadline,
+                )
+            except exceptions.NotFound:
+                logger.exception("Cannot subscribe to a topic that does not exist.")
```

Comment:
Since we are handling a very specific exception and case, we can use ERROR logging level instead of exception here. I'd also add the topic name as an interpolated variable to the log message